main.go

```diff
@@ -93,26 +33,21 @@ func main() {
 
 	if c.Debug {
 		logrus.SetLevel(logrus.DebugLevel)
-		_ = server.Server.Flags().Set("log-level", "debug")
 	}
 
-	// Yeah, not a good practice at all but I guess it's fine for now.
-	proxy.ActualDefaultRequestHandler.RequestKeyHandlers.Set(kafka.RequestAPIKeyProduce, &requestKeyHandler{})
-
-	for _, v := range c.KafkaProxyExtraFlags.values {
-		f := strings.Split(v, "=")
-		_ = server.Server.Flags().Set(f[0], f[1])
-	}
-
-	for _, v := range c.KafkaProxyBrokersMapping.values {
-		_ = server.Server.Flags().Set("bootstrap-server-mapping", v)
+	proxyConfig := kafka.ProxyConfig{
```

Comment:
```suggestion
	kafkaProxyConfig := kafka.ProxyConfig{
```

As I understand it's not the config for all protocols :)

Reply:
The name is inferred from the package. In fact some linters will complain most probably if I add the `kafka` prefix to a type from a package of the same name.